test/system/markdown_test.rb

```diff
@@ -13,6 +13,7 @@ class MarkdownTest < ApplicationSystemTestCase
     assert_css '.a-long-text.is-md.js-markdown-view'
     assert_css '.speak'
     assert_css "a[href='/users/mentormentaro']"
+    assert_includes find('.js-user-icon.a-user-emoji')['title'], '@mentormentaro'
     assert_includes find('.js-user-icon.a-user-emoji')['data-user'], 'mentormentaro'
```

Comment:
```suggestion
    emoji = find('.js-user-icon.a-user-emoji')
    assert_includes emoji['title'], '@mentormentaro'
    assert_includes emoji['data-user'], 'mentormentaro'
```

`.js-user-icon.a-user-emoji`というクラスが変わるかもしれないということを考えてこんな感じにDRYにするのも良いかもです。

Reply:
@komagata
レビューありがとうございます！
その方がすっきり見えますし、変更にも強いテストコードになりそうですね！
修正しようと思います。

ちなみに下の方のassert_includes find('.js-user-icon.a-user-emoji')['data-user'], 'mentormentaro'という文は元からあったコードなのですが、このPR内で修正してもよいでしょうか？

Reply:
@a-kuroki-gs はい、元のコードも含めてベストな状態に変更するようにしてください〜

Reply:
@komagata
ありがとうございます！
ご指摘いただいたテストの内容を修正しました。
確認よろしくお願いいたします。

Reply:
@a-kuroki-gs 
お忙しいところすみません！
ほんとにお時間ある時で大丈夫なので、今回のコミット履歴を直した手順をご共有頂いてもよろしいでしょうか…。
実務も忙しいと思うので、このメンションへの返信もなしで大丈夫です🙇‍♂️

いきなりのメンション失礼いたしました！

Reply:
@komagata 
> コミットは綺麗な理想的な状態にしてPRをお願いします。
仕事が始まると特に注意されちゃう箇所かなと思います。

そうですよね…ご指摘ありがとうございます。
コミットログが正しくなるよう修正しましたので、確認よろしくお願いします！

---

@masyuko0222 
色々と通知飛ばしてしまい申し訳ないです🙏

今日の日報に書こうと思ってまとめていたところでした！
まだWIP状態ですが、手順記載しておりますのでよろしければご確認ください🙌
https://bootcamp.fjord.jp/reports/89693